src/javascripts/components/cookies-page.js

```diff
@@ -0,0 +1,174 @@
+import { getConsentCookie, setConsentCookie } from '../cookie-functions.js'
+import common from 'govuk-frontend/govuk/common'
+
+const nodeListForEach = common.nodeListForEach
+
+function CookiesPage ($module) {
+  this.$module = $module
+}
+
+CookiesPage.prototype.init = function () {
+  this.$cookiePage = this.$module
+
+  if (!this.$cookiePage) {
+    return
+  }
+
+  this.$cookieForms = this.$cookiePage.querySelectorAll('.js-cookies-page-form-wrapper')
+  this.$errorSummary = this.$cookiePage.querySelector('.js-cookies-page-error-summary')
+  this.$errorSummaryListItems = this.$errorSummary.querySelectorAll('.govuk-error-summary__list a')
+  this.$saveButton = this.$cookiePage.querySelector('.js-cookies-page-save')
+  this.$successNotification = this.$cookiePage.querySelector('.js-cookies-page-success')
+
+  nodeListForEach(this.$cookieForms, function ($cookieFormGroup) {
+    this.showUserPreference($cookieFormGroup, getConsentCookie())
+    $cookieFormGroup.removeAttribute('hidden')
+  }.bind(this))
+
+  this.$saveButton.addEventListener('click', this.savePreferences.bind(this))
+  this.$saveButton.removeAttribute('hidden')
+}
+
+CookiesPage.prototype.savePreferences = function () {
+  var preferences = {}
+
+  nodeListForEach(this.$cookieForms, function ($cookieForm) {
+    var cookieType = this.getCookieType($cookieForm)
+    var radios = $cookieForm.elements[cookieType]
+    var selectedItem = radios.value
+
+    if (!selectedItem) {
+      this.showErrorMessage($cookieForm)
+      return
+    }
+
+    this.hideErrorMessage($cookieForm)
+    preferences[cookieType] = radios.value === 'yes'
```

Comment:
Given we've assigned `radios.value` to `selectedItem` above, I think it'd make sense to use it here?

```suggestion
    preferences[cookieType] = selectedItem === 'yes'
```